File: Data/DataTransferObjects/DocumentationResponseDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Fr8Data.DataTransferObjects
{
    public class DocumentationResponseDTO
    {
        public string Name { get; set; }
        public double Version { get; set; }
        //TODO: To be changed with another type
        public string Terminal { get; set; }
        //This field is to hold an HTML
        public string Body { get; set; }
    }

}

File: Scripts/app/model/DocumentationResponseDTO.ts
module dockyard.model {
	export class DocumentationResponseDTO {
        name: string;
        version: string;
        terminal: string;
        body: string;
        activityName: string;

        constructor(name: string, version: string, terminal: string, body: string, activityName: string) {
            this.name = name;
            this.version = version;
            this.terminal = terminal;
            this.body = body;
            this.activityName = activityName;
        }
    }
}